src/content/blog/2024/nxf-nf-core-workshop-kogo.md

```diff
@@ -9,6 +9,7 @@ status: published
 author: Yuk Kei Wan
 icon: yukkei.jpg
 community_post: true
+ambassador_post: true
```

Comment:
```suggestion
```

She's not an ambassador 😅. We keep thinking she is, but she said she's too busy 😅

Reply:
Also, it'd be good to have the `ambassador_post` tag automatically added. Do you think it's possible?

Reply:
I don't know how we would add this automatically - we would have to have a central list of ambassador identities somewhere and cross-reference the author against this or something I guess?

Sounds like quite a lot of work and quite error prone, so I would prefer to keep it like this unless I'm missing something.